app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable" : [my_favourite_films, marks, collection_of_coins],
+    "immutable" : [lucky_number, pi, one_is_a_prime_number, name, profile_info]
+}
```

Comment:
The `collection_of_coins` is categorized as mutable, but sets in Python are mutable only in terms of their elements being changeable (i.e., you can add or remove elements). However, the set itself cannot be changed in terms of its identity, which is why it is often considered immutable in certain contexts. If the task requires strict immutability, consider this aspect.